filebeat/input/log/harvester.go

```diff
@@ -276,6 +276,10 @@ func (h *Harvester) Run() error {
 				logp.Info("End of file reached: %s. Closing because close_eof is enabled.", h.state.Source)
 			case ErrInactive:
 				logp.Info("File is inactive: %s. Closing because close_inactive of %v reached.", h.state.Source, h.config.CloseInactive)
+			case reader.ErrLineUnparsable:
+				logp.Err("Skipping unparsable line in file: %v", h.state.Source)
```

Comment:
Nit: I think this log message should be Info, as we have already emitted an error message from `DockerJSON` with the error message. To me this is rather a notification telling the user that a line was skipped due to an error before.

Reply:
I changed it to info

Reply:
I think you have changed the wrong line for the case `default`.

Reply:
this time, I changed what i was supposed to change ;)